### ISSUE TITLE
Accent color got back to default value after deleting a key

### ISSUE THREAD

**Issue by netzego** [orhun/gpg-tui#51]
Accent color got back to default value after deleting a key.

**Steps to reproduce the behavior**

1. start `gpg-tui`
2. select a pub key
3. press `d` and `y`

``` ~/.config/gpg-tui/config
[general]
splash = false
tick_rate = 250
color = "blue"
style = "plain"
file_explorer = "xplr"
detail_level = "minimum"
key_bindings = [
  { keys = [
    "?",
    "h",
    "f1",
  ], command = ":help" },
  { keys = [
    "C-d",
    "C-c",
    "q",
  ], command = ":quit" },
]

[gpg]
armor = false
homedir = "~/.gnupg"
outdir = "~/.gnupg/out"
outfile = "{type}_{query}.{ext}"
default_key = ""
```

**System Information**

 - OS: Arch Linux 6.0.17-hardened1-1-hardened
 - Rust Version: rustc 1.64.0-nightly (6dbae3ad1 2022-07-25)
 - gpg (GnuPG) 2.2.40
 - gpg-tui 0.9.2 (paru -S pgp-tui-git)

**Comment by orhun** [comment]
Hey!

This should be fixed in 6ecc253c5c54c2779b991170eed4ff03ee539106

Thanks for reporting! 🐻

**Comment by netzego** [comment]
Hey you, can confirm that :) thx, for your work and have a nice day!
